spec/element_base_spec.rb

```diff
@@ -87,6 +87,24 @@
     end
   end
 
+  describe '#add_ancestor' do
```

Comment:
I only did `#add_ancestor` and not `#add_ancestors` because I thought that since in all cases, `#add_ancestors` uses `#add_ancestor`, a separate spec for `#add_ancestors` isn't necessary. Is this correct? @jpslav